src/Lykke.AzureStorage/IBlobStorage.cs

```diff
@@ -39,8 +39,37 @@ public interface IBlobStorage
 
         string GetBlobUrl(string container, string key);
 
+        [Obsolete("This method requires Uri to be present in prefix. Use GetListOfBlobKeysByPrefixAsync instead.")]
         Task<IEnumerable<string>> FindNamesByPrefixAsync(string container, string prefix);
 
+        /// <summary>
+        ///     Delete all blobs starting with <paramref name="prefix" />.
+        ///     This method is useful for deleting folders.
+        ///     Specify folder as prefix and all blobs and folders inside it would be deleted.
```

Comment:
better change 'inside it' -> 'inside container' to avoid misunderstanding

Reply:
done.